tests/integration/Entries/GetEdocWithAuditReasonTest.cs

```diff
@@ -32,31 +32,10 @@ public async Task Cleanup()
             }
         }
 
-        private async Task<int> CreateDocument()
-        {
-            int parentEntryId = 1;
-            string fileName = "RepositoryApiClientIntegrationTest .Net GetDocumentContent AuditReason";
-            string fileLocation = TempPath + "test.pdf";
-            var request = new PostEntryWithEdocMetadataRequest();
-            using (var fileStream = File.OpenRead(fileLocation))
-            {
-                var electronicDocument = new FileParameter(fileStream, "test", "application/pdf");
-                var result = await client.EntriesClient.ImportDocumentAsync(RepositoryId, parentEntryId, fileName, autoRename: true, electronicDocument: electronicDocument, request: request);
-
-                var operations = result.Operations;
-                Assert.IsNotNull(operations?.EntryCreate);
-                Assert.AreEqual(0, operations.EntryCreate.Exceptions.Count);
-                Assert.AreNotEqual(0, operations.EntryCreate.EntryId);
-                Assert.AreEqual(0, operations.SetEdoc.Exceptions.Count);
-                Assert.IsFalse(string.IsNullOrEmpty(result.DocumentLink));
-                return operations.EntryCreate.EntryId;
-            }
-        }
-
         [TestMethod]
         public async Task Export_document_with_audit_reasonAsync_ReturnDocument()
         {
-            createdEntryId = await CreateDocument();
+            createdEntryId = await CreateDocument("RepositoryApiClientIntegrationTest .Net ExportDocumentWithAuditReasonAsync");
```

Comment:
yes, previously we were creating a test document in the repository with that name, but i changed it to match the name of the function being called. But since we don't seem to follow that pattern in other tests, I changed it back now.